FirebaseFunctions/Sources/Callable+Codable.swift

```diff
@@ -12,154 +12,102 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import Foundation
-import FirebaseSharedSwift
+import Foundation import FirebaseSharedSwift
 
-// A `Callable` is reference to a particular Callable HTTPS trigger in Cloud Functions.
-public struct Callable<Request: Encodable, Response: Decodable> {
```

Comment:
Please update the PR to be just the new API implementation and I'll take another look